### PR TITLE
Fix hidden long text in output

### PR DESCRIPTION
**Pull request by merelj**
## Description of the PR
Columns inherited css-style `overflow: hidden` from html. I added class  `is-horizontally-scrollable` to override this.

### Issues Related
[Long text in output is not visible #348](https://github.com/angrykoala/gaucho/issues/348)

------

Please, before doing the PR make sure that you are following the intructions described in [CONTRIBUTING.md](https://github.com/angrykoala/gaucho/blob/master/CONTRIBUTING.md) and you have completed the following checklist

**Checklist**
- [ ] Your PR is done against the `dev` branch
- [ ] Your origin branch is updated with the latest changes from `dev`
- [ ] All the tests and linters are passing (`npm run test`)
- [ ] Your changes have tests

Thanks for contributing to Gaucho

### CODE REVIEW

**Comment by angrykoala** [comment]
Hi @merelj 
Thanks for your PR, everything in the code looks fine and works as expected, however, there is a minor style bug in which the horizontal scroll appears under a white padding:

File: src/app/components/common/text_output.vue [hunk 1]
```diff
@@ -46,14 +46,14 @@ module.exports = {
...
-    overflow-y: auto;
-    overflow-x: hidden;
+    overflow: auto;
     height: 200px;
     border-bottom-style: solid;
     border-bottom-width: 1px;
     border-color: #e2e2e2;
     pre {
-        overflow: hidden;
+        width: fit-content;
```
Comment (merelj) [inline src/app/components/common/text_output.vue]: this won't work in IE and Edge, so next line is fallback for them

**Comment by merelj** [comment]
@angrykoala fair point, I've decided to go with another approach. hope it looks better 🤞

File: src/app/components/common/text_output.vue (hunk 1 above)
Comment (angrykoala) [reply src/app/components/common/text_output.vue]: Not really needed, as electron runs on chrome but it's a nice touch

**Comment by angrykoala** [comment]
It seems to be working fine, thanks @merelj

**Pull request merged**